server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesAction.java

```diff
@@ -32,6 +34,6 @@ private FieldCapabilitiesAction() {
 
     @Override
     public FieldCapabilitiesResponse newResponse() {
-        return new FieldCapabilitiesResponse();
+        return new FieldCapabilitiesResponse(Collections.emptyMap());
```

Comment:
Wasn't this one for serialization? I would have changed the other call from the TransportAction, this is ok as-is I think, unless we want to completely get rid of the default constructor.

Reply:
oups, I fixed, thanks !